debian/clearwater-infrastructure.links

```diff
@@ -8,5 +8,5 @@
 /usr/share/clearwater/bin/clearwater-show-config /usr/sbin/clearwater-show-config
 /usr/share/clearwater/bin/clearwater-check-config /usr/sbin/cw-check_config
 /usr/share/clearwater/bin/clearwater-show-config /usr/sbin/cw-show_config
-/usr/share/clearwater/infrastructure/bin/set_snmp_community /usr/bin/cw-set_snmp_community
-/usr/share/clearwater/infrastructure/bin/set_log_level /usr/bin/cw-set_log_level
+/usr/share/clearwater/infrastructure/bin/set_snmp_community /usr/sbin/cw-set_snmp_community
```

Comment:
Shouldn't an admin be able to use change the SNMP community string?

Reply:
Yes, but they'll need to use the `run_set_snmp_community` script because it requires sudo elevation. We'll be adding the link for that in the clearwater-core only code.